literature.bib

```diff
@@ -3146,6 +3242,76 @@ @inproceedings{FMS+:SANER17
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2016 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{ZRL:VAMOS16,
+	author = {Ziegler, Andreas and Rothberg, Valentin and Lohmann, Daniel},
+	title = {{Analyzing the Impact of Feature Changes in Linux}}, 
+	booktitle = VaMoS,
+	year = 2016,
+	pages = {25--32},
+	publisher = ACM,
+	address = NY,
+	url = {https://doi.org/10.1145/2866614.2866618},
+	doi = {10.1145/2866614.2866618},
+	numpages = {8},
+	keywords = {Kconfig, Configurability, CADOS, Linux},
+	location = {Salvador, Brazil},
+} 
+
+@inproceedings{PMK+:GPCE16,
+	author = {Pereira, Juliana Alves and Matuszyk, Pawel and Krieter, Sebastian and Spiliopoulou, Myra and Saake, Gunter},
+	title = {{A Feature-Based Personalized Recommender System for Product-Line Configuration}}, 
+	booktitle = GPCE,
+	year = 2016,
+	publisher = ACM,
+	address = NY,
+	url = {https://doi.org/10.1145/2993236.2993249},
+	doi = {10.1145/2993236.2993249},
+	numpages = {12},
+	keywords = {Software Product Lines, Personalized Recommendations, Recommenders, Product-Line Configuration},
+	location = {Amsterdam, Netherlands},
+} 
+
+% How to get the address here?
```

Comment:
proceed as discussed

Reply:
What was the discussion result? Should we add it to our DOs and DON'Ts? https://github.com/SoftVarE-Group/BibTags/wiki

Reply:
We do not remember. Changing the note to "Could not find address" to make it consistent with other entries where that happened.